trecs/models/bass.py

```diff
@@ -46,57 +49,119 @@ def __init__(self, infection_thresholds=None, verbose=False):
 
 
 class BassModel(BaseRecommender, BinarySocialGraph):
```

Comment:
We didn't have documentation for `BassModel` so I took a stab at adding some while I was editing other things